server.js

```diff
@@ -57,7 +57,7 @@ const authenticate = function(request, response, next) {
 
     let requestToken = request.body.token;
 
-    if (!requestToken) {
+    if (config.globalConf.authToken && !requestToken) {
```

Comment:
mhh i dont like implicit assumptions. leaving something out or forgetting something leading to an open app.

i would prefer an explicit value like 'DISABELED_AUTH_CHECK' or a own setting.

Reply:
Sure, I'll change to something more explicit then.